jackson-it.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -e
+mvn install -DskipTests=true
+for v in $(curl -s 'http://search.maven.org/solrsearch/select?q=g:%22com.fasterxml.jackson.core%22+AND+a:%22jackson-databind%22&core=gav&rows=100&wt=json' |
+           jq -r '[.response.docs | .[].v |
+                  {p: split("."), v:.} |
+                  {major:.p[0] | tonumber, minor: .p[1] | tonumber, v:.v} |
+                  select(.major >= 2 and .minor >= 4)] |
+                  sort_by(.v | split(".")) |
```

Comment:
Here, the versions will not be sorted correctly. You probably want a `tonumber` here